docs/styles/spec/file-input-spec.less

```diff
@@ -1,5 +1,5 @@
 @import (reference) 'components/button/index';
```

Comment:
@lalithkarikelli, please revert changes to this file and remove from this PR.  We will remove with PR #650.

Reply:
Changes have been done, please review.